meshnets/main.py

```diff
@@ -1,45 +1,65 @@
 """The main file to run our code
 
-Currently, it loads mesh data from a .vtk file, defines a wind vector,
-and creates a Graph object from them
+Currently, it processes mesh data to graph data and creates training
```

Comment:
I think we should rename this file according to its function

Reply:
If this file will process mesh data to graph data we could rename it to something like: ~data_preprocessing.py~ or ~mesh_to_graphs.py~

Reply:
test_pipeline.py?

Reply:
I moved the processing loop to the processing.py file so the main method is now only a sequence of very simple steps. I think we can keep the name for now. The pipeline will certainly change again and main.py does not have a well defined role for now.